packages/astro/src/@types/astro.ts

```diff
@@ -706,6 +706,27 @@ export interface AstroUserConfig {
 	 * @name Image options
 	 */
 	image?: {
+		/**
+		 * @docs
+		 * @name image.service (Experimental)
+		 * @type {'astro/assets/services/sharp' | 'astro/assets/services/squoosh' | string}
+		 * @default `'astro/assets/services/squoosh'`
+		 * @version 2.1.0
+		 * @description
+		 * Set which image service is used for Astro’s experimental assets support.
+		 *
+		 * The value should be a module specifier for the image service to use:
+		 * either one of Astro’s two built-in services, or a third-party implementation.
+		 *
+		 * ```js
+		 * {
+		 *   image: {
+		 *     // Example: Enable the Sharp image processor
```

Comment:
```suggestion
		 *     // Example: Enable the Sharp-based image service
```
Maybe? I think there's an important distinction between "image processor" and "image service" since we document image services as a concept:tm: